.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
IDE related ignore rules should be part of your global git config within your development environment

Reply:
Thx. Fixed